maestro-android/build.gradle

```diff
@@ -144,6 +144,9 @@ dependencies {
     implementation(libs.ktor.server.content.negotiation)
     implementation(libs.ktor.serial.gson)
 
+    implementation('org.apache.commons:commons-lang3:3.13.0')
+    implementation('org.lsposed.hiddenapibypass:hiddenapibypass:4.3')
```

Comment:
What does this library do?

Reply:
since API level 30 `updateConfiguration` API was hidden and is not accessible anymore via `Class.getMethod` API. This library allows to access hidden APIs. Under the hood the library uses Java `Unsafe` API, which can read and write memory in pure Java.

more info [here](https://lovesykun.cn/archives/android-hidden-api-bypass.html)